packages/amplify-codegen/tests/commands/__snapshots__/statements-mock-fs.test.js.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`command - types (mock fs) should generate statements for android - android 1`] = `
+Array [
+  "mutations.graphql",
+  "queries.graphql",
+  "subscriptions.graphql",
+]
+`;
+
```

Comment:
Instead of snapshots, could you embed the config input and outputs into the test config, where you've got that mapping of frontend + target, just embed the expected result, instead of decoupling into a snapshot file. I think it'll be easier to reason about how these things map together.